pkgs/by-name/kr/kryptor/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+}:
+
+buildDotnetModule rec {
+  pname = "kryptor";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "samuel-lucas6";
+    repo = "Kryptor";
+    rev = "v${version}";
+    hash = "sha256-BxUmDzmfvRelQDHb5uLcQ2YPL7ClxZNFGm/gQoDK8t8=";
+  };
+
+  projectFile = "src/Kryptor.sln";
+  nugetDeps = ./deps.nix;
+
+  executables = ["kryptor"];
+
+  packNupkg = true;
+
+  patches = [
+    ./include-icon.patch
+  ];
+
+  meta = with lib; {
+    description = "A simple, modern, and secure encryption and signing tool that aims to be a better version of age and Minisign";
+    homepage = "https://github.com/samuel-lucas6/Kryptor";
+    changelog = "https://github.com/samuel-lucas6/Kryptor/releases/tag/v${version}";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ arthsmn ];
+    mainProgram = "kryptor";
+    platforms = platforms.all;
```

```suggestion
  meta = {
    changelog = "https://github.com/samuel-lucas6/Kryptor/releases/tag/v${version}";
    description = "A simple, modern, and secure encryption and signing tool that aims to be a better version of age and Minisign";
    homepage = "https://github.com/samuel-lucas6/Kryptor";
    license = lib.licenses.gpl3Only;
    maintainers = with lib.maintainers; [ arthsmn ];
    mainProgram = "kryptor";
    platforms = lib.platforms.all;
```

A small nit using `with lib;` on `meta` is discouraged, as well as I sorted it alphabetically

---

Done. Thanks!